vowpalwabbit/parse_args.h

```diff
@@ -31,3 +31,4 @@ void parse_sources(VW::config::options_i& options, vw& all, io_buf& model, bool
 VW::LEARNER::base_learner* setup_base(VW::config::options_i& options, vw& all);
 
 std::string spoof_hex_encoded_namespaces(const std::string& arg);
+bool ends_with(const std::string& fullString, const std::string& ending);
```

Comment:
If we want to cheat a little, we could actually make the input parameters `VW::string_view`, and then call endwith on those ;)
Boost::StringRef and Boost::string_view both implement starts_with() and ends_with

Reply:
That sounds kind of nice - but for now I am just exposing the functions in the header and don't want to change the impl